database.rules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    ".read": "auth != null",
```

Comment:
You'll want to limit this such that users can only read/write their own data eventually. (open an issue?)